packages/filebrowser/src/listing.ts

```diff
@@ -2111,7 +2111,7 @@ export namespace DirListing {
     /**
      * Create a node for a header item.
      */
-    private _createHeaderItemNode(label: string): HTMLElement {
+    protected _createHeaderItemNode(label: string): HTMLElement {
```

Comment:
If we decide to make this `protected`, we should remove the `_` from the name; that is strictly for `private` methods and attributes.

Reply:
Thanks for the review!